content/workshops/example/workshop_content.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Workshop Content
+workshops: example
+workshop_weight: 30
+layout: lab
+---
+
+## SETTING EXPECTATIONS FOR YOUR AUDIENCE
+
+----
+
+- Clearly tell your audience what the lab is, what things they will be doing, and what kind of skills are required to be successful at your workshop.
+
+- Examples of expectation setting:
+	- “This is a hands-on workshop.”
+	- “You should be comfortable with the command line.”
+	- “You should be comfortable being a Java developer.”
+	- “You understand classes and how to create them.”
+	- “You can write, understand and create html/jsp/servlets.”
```

Comment:
Each of the items in this list have smart quotes instead of regular quotes. That tends to cause undesired behavior in browsers. Change to regular quotes.